dcm4ceph-core/src/main/java/org/open_ortho/dcm4ceph/core/Cephalogram.java

```diff
@@ -107,12 +107,18 @@ public Cephalogram(File cephFile, File config) {
         init();
 
         if (config == null) {
```

Comment:
Default should be looking for a properties file with same name as filename, with extension swapped with .properties. If that fails, then printing the message instructing user how to create properties file is appropriate. 

The reasoning is that you should not be able to generate a DICOM file without clinically relevant and mandatory DICOM tags. Remove and/or correct the comments in this section as appropriate

Reply:
If I understand correctly, the option with explicit config file is to be removed.
Basically one and only way is the .properties file with swapped name. Just for overall simplicity.

Reply:
No, what i meant was that the code first checked for the CLI argument list: if properties is there, use that. If it is not, use the one with `.properties` file extension. If that fails, `Log.error` with instructions on how to create and use `.properties` files.

Reply:
Returned the logic back, though CLI arg is for the properties not here. So I kinda decided this time not to introduce it unless told to do so :)